src/Evolution/DiscontinuousGalerkin/Actions/PackageDataImpl.hpp

```diff
@@ -53,13 +53,34 @@ double dg_package_data(
         unit_normal_covector,
         get<evolution::dg::Actions::detail::NormalVector<Dim>>(
             projected_fields),
-        mesh_velocity, normal_dot_mesh_velocity, db::get<VolumeTags>(box)...);
+        mesh_velocity, normal_dot_mesh_velocity, volume_args...);
   } else {
     return boundary_correction.dg_package_data(
         make_not_null(&get<PackagedFieldTags>(*packaged_data))...,
         get<ProjectedFieldTagsForCorrection>(projected_fields)...,
         unit_normal_covector, mesh_velocity, normal_dot_mesh_velocity,
-        db::get<VolumeTags>(box)...);
+        volume_args...);
   }
 }
+
+template <typename System, typename BoundaryCorrection,
+          typename... PackagedFieldTags, typename... ProjectedFieldTags,
+          typename... ProjectedFieldTagsForCorrection, size_t Dim,
+          typename DbTagsList, typename... VolumeTags>
+double dg_package_data(
+    const gsl::not_null<Variables<tmpl::list<PackagedFieldTags...>>*>
+        packaged_data,
+    const BoundaryCorrection& boundary_correction,
+    const Variables<tmpl::list<ProjectedFieldTags...>>& projected_fields,
+    const tnsr::i<DataVector, Dim, Frame::Inertial>& unit_normal_covector,
+    const std::optional<tnsr::I<DataVector, Dim, Frame::Inertial>>&
+        mesh_velocity,
+    const db::DataBox<DbTagsList>& box, tmpl::list<VolumeTags...> /*meta*/,
+    tmpl::list<ProjectedFieldTagsForCorrection...> /*meta*/) noexcept {
+  return dg_package_data<System>(
```

Comment:
In the `internal_mortar_data` case, you renamed the non-Databox version with an `_impl` suffix. Should the same be done here? Or am I missing something about the interfaces (e.g., is the Databox interface for `dg_package_data` still called directly?)?

Reply:
The DataBox interface is still called in the boundary conditions at this point. I haven't finalized that code yet. I'm hoping to be able to get rid of the `DataBox` one completely, if possible. Since they both currently act as a semi-public interface, or at least are on equal footing in terms of whether they should be called directly, I decided to not add `_impl`. Does that sound okay?